public/apps/configuration/panels/tenant-list/tenant-list.tsx

```diff
@@ -59,8 +58,8 @@ export function TenantList(props: TenantListProps) {
     <>
       <EuiCallOut title="Tenancy is disabled" color="warning" iconType="iInCircle">
         <p>
-          Tenancy is currently disabled and users don&apos;t have access to this feature. To create,
-          edit tenants you must enabled tenanc throught he configure tenancy page.
+          Tenancy is currently disabled and users don&apos;t have access to this feature. Enable
+          tenancy through the configure tenancy page.
```

Comment:
nit: why a new line mid sentence

Reply:
The linter automatically does it if the line gets too long.